inst/project/blueprints/example/homogenized.csv

```diff
@@ -0,0 +1,14 @@
+name,type,description,coding
+name,character,Name of Car,
+mpg,double,Miles per gallon,
+cyl,double,Number of cylinders,
+disp,double,Displacement,
+hp,double,Gross horsepower,
+drat,double,Rear axle ratio,
+wt,double,Weight,
+qsec,double,Quarter mile time,
+vs,character,Engine,"coding(code(""1"",""straight""), code(""0"", ""v-shaped""))"
+am,character,Transmission,"coding(code(""1"",""manual""), code(""0"", ""automatic""))"
```

Comment:
I think the codings on these are backwards

Reply:
They're automatically coded in the `mtcars` dataset as `0`s and `1`s, but wanted to give an example of how coding could work. Should I keep as is?

Reply:
@psanker let me know! I haven't changed this yet, but can! Wish `mtcars` didn't already have everything coded.

Reply:
Oh sorry for the delay. I mean the label should come first then the value, e.g. `coding(code("Straight", 1), code("V-shaped", 0))`

Reply:
Ohhh I see. Ok fixing rn